### PR TITLE
Added all website files

### PR DESCRIPTION
**Pull request by ElenaLadutska**
Created dev branch and pushed all website files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,446 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+  <title>Dashboard</title>
+</head>
+<body>
+  <div class="container">
+
```
Comment (kathrynowy) [inline index.html]: unnecessary line

File: index.html [hunk 2]
```diff
@@ -0,0 +1,446 @@
...
+            <img src="./assets/header-icons/notifications.png" alt="Notification button">
+          </div>
+        </div>
+      </header>
+
+      <div class="content">
+        <div class="statistics-container">
+          <div class="full-online-statics">
+            <div class="online-statics">
+
```
Comment (kathrynowy) [inline index.html]: unnecessary line

File: index.html [hunk 3]
```diff
@@ -0,0 +1,446 @@
...
+                  <div class="online-display-of-the-indicator attendantness"></div>
+                  <div></div>
+                </div>
+
+                <p class="percents-of-the-indicator">80%</p>
+              </div>
+            </div>
+
+            <div class="online-statics">
+
```
Comment (kathrynowy) [inline index.html]: same
unnecessary line

we don't paste empty lines when we OPEN a tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,446 @@
...
+          </div>
+        </div>
+      </footer>
+    </div>
+
+  </div>
+</body>
+</html>
+
+<!-- Overflow elipsis -->
```
Comment (kathrynowy) [inline index.html]: eof

Comment (kathrynowy) [reply index.html]: remove unnecessary comment

File: reset.css [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
```
Comment (kathrynowy) [inline reset.css]: eof

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,733 @@
...
+
+:root {
+  --gondola:#090606;
+  --cod-gray: #0E0D0D;
+  --hint-of-red:#F5F0F1;
+  --gradient-radical-red-and-red-ribbon:linear-gradient(#FF3F58 100%, #E50D29 100%);
+  --light-gray: #F5F0F11A;
+  --radical-red: #FF3F58;
+  --dodger-blue:#18B4F9;
+  --yellow-orange:#FEA441;
```
Comment (kathrynowy) [inline styles.css]:
```suggestion
  --dodger-blue: #18B4F9;
  --yellow-orange: #FEA441;
```

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (ElenaLadutska) [reply index.html]: fixed

File: index.html (hunk 2 above)
Comment (ElenaLadutska) [reply index.html]: fixed

File: index.html (hunk 3 above)
Comment (ElenaLadutska) [reply index.html]: fixed

File: index.html (hunk 4 above)
Comment (ElenaLadutska) [reply index.html]: fixed

File: reset.css (hunk 5 above)
Comment (ElenaLadutska) [reply reset.css]: fixed

File: styles.css (hunk 6 above)
Comment (ElenaLadutska) [reply styles.css]: fixed

**Review by kathrynowy — APPROVED** [review]

**Pull request merged**